packages/ui/core-components/src/lib/atoms/inputs/button-group/ButtonGroupItem.svelte

```diff
@@ -20,16 +25,26 @@
 	}
 </script>
 
-<button
-	type="button"
-	class=" flex-none py-1 font-medium h-8 px-3 text-xs truncate
-                                border-r last:border-none
-                                hover:bg-gray-100 focus:z-10 focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-gray-400
-                                {$currentValue?.value === value
-		? 'z-10 border-gray-200 bg-gray-100 text-blue-700'
-		: 'z-0 bg-white text-gray-900 border-gray-200'}
+{#if display === 'tabs'}
+	<TabDisplay
+		id={value}
+		label={valueLabel}
+		{color}
+		on:click={() => update({ valueLabel, value })}
+		activeId={$currentValue?.value}
+	/>
+{:else}
+	<button
+		type="button"
+		class=" flex-none py-1 font-medium h-8 px-3 text-xs truncate
```

Comment:
is there a reason we can't do the tab styling with just classes?

rather than a new component for the tab display

feels like it would be easier to maintain this way

Reply:
By having a display tabs component we can maintain consistency between tabs and buttons styled as tabs in one component. Without that, the two styles could diverge over time.

Reply:
I am more concerned about divergence of _functionality_ between buttonGroup styled as tabs and ButtonGroup styled as buttons